db/db_test_util.cc

```diff
@@ -671,6 +671,21 @@ std::string DBTestBase::Get(int cf, const std::string& k,
   return result;
 }
 
+std::string DBTestBase::Get(const std::string& k, PinnableSlice* v) {
+  ReadOptions options;
+  options.verify_checksums = true;
+  std::string result;
+  Status s = dbfull()->Get(options, dbfull()->DefaultColumnFamily(), k, v);
+  if (s.IsNotFound()) {
+    result = "NOT_FOUND";
+  } else if (!s.ok()) {
+    result = s.ToString();
```

Comment:
test cases for Get calls in db_test.cc are validated using ASSERT_EQ statements, which checks if the return value is same as the value expected. So result of DBTestBase::Get() returns actual value in case of success and returns status code strings incase of failure. If the code is confusing and looks mixed up, I can change it to 
{
.
.
Status s = dbfull()->Get(options, dbfull()->DefaultColumnFamily(), k, v);
std::string result(v->data());
  if (s.IsNotFound()) {
    result = "NOT_FOUND";
  } else if (!s.ok()) {
    result = s.ToString();
  }
  return result;
}

which is exactly same as which is exactly same as std::string DBTestBase::Get(int cf, const std::string& k,
                            const Snapshot* snapshot)  in db_test_util.cc